[2023-04-14T22:13:52.057+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pads.launchpad scheduled__2023-01-01T00:10:00+00:00 [queued]>
[2023-04-14T22:13:52.194+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pads.launchpad scheduled__2023-01-01T00:10:00+00:00 [queued]>
[2023-04-14T22:13:52.202+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T22:13:52.211+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2023-04-14T22:13:52.217+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T22:13:53.027+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): launchpad> on 2023-01-01 00:10:00+00:00
[2023-04-14T22:13:53.067+0000] {standard_task_runner.py:55} INFO - Started process 376 to run task
[2023-04-14T22:13:53.095+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pads', 'launchpad', 'scheduled__2023-01-01T00:10:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpuziijjb3']
[2023-04-14T22:13:53.104+0000] {standard_task_runner.py:83} INFO - Job 23: Subtask launchpad
[2023-04-14T22:13:54.081+0000] {task_command.py:389} INFO - Running <TaskInstance: pads.launchpad scheduled__2023-01-01T00:10:00+00:00 [running]> on host a0b8dfb9f67f
[2023-04-14T22:13:54.929+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Amogh Srigopal
AIRFLOW_CTX_DAG_ID=pads
AIRFLOW_CTX_TASK_ID=launchpad
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:10:00+00:00
[2023-04-14T22:13:54.943+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-14T22:13:54.951+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/amoghsrigopal/Downloads/SpaceDataPipeline/launchpad.py']
[2023-04-14T22:13:55.020+0000] {subprocess.py:86} INFO - Output:
[2023-04-14T22:13:56.044+0000] {subprocess.py:93} INFO - python: can't open file '/Users/amoghsrigopal/Downloads/SpaceDataPipeline/launchpad.py': [Errno 2] No such file or directory
[2023-04-14T22:13:56.057+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-04-14T22:13:56.102+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-04-14T22:13:56.154+0000] {taskinstance.py:1332} INFO - Marking task as UP_FOR_RETRY. dag_id=pads, task_id=launchpad, execution_date=20230101T001000, start_date=20230414T221352, end_date=20230414T221356
[2023-04-14T22:13:56.328+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 23 for task launchpad (Bash command failed. The command returned a non-zero exit code 2.; 376)
[2023-04-14T22:13:56.510+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-14T22:13:58.502+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
